### ISSUE TITLE
Calendar Accessibility

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#170]
## Description of issue(s)

1. Previous/Next month buttons don't have voiceover labels

The following platform-specific issues may just be down to the differences between the Android/iOS text-to-speech tools, but it would be nice to have some consistency between their behaviours. 

iOS:
2. Selecting a date (by double-tapping) reads out the selection, then the date button label e.g. "Wednesday the 9th of May, selected. Wednesday the 9th of May, button."
3. Doesn't read out the selection range like it does on Android (e.g. "Selected, 10th July minus 12th July")

Android:
4. Doesn't tell you if a button is selected when you tap on it, like iOS does (e.g. "Selected, Thursday the 10th July) 

## Expected behaviour

1. There is a voiceover label for each of the Previous/Next month buttons.
2. When selecting a date it should just read out the selection, e.g. "Selected, Wednesday the 9th of May"
3. When a range is selected on iOS it should read out the selected range, e.g. "Selected, Thursday the 10th of July to Saturday the 12th of July"
4. When tapping on an already selected date it should read out that it is selected, e.g. "Selected, Thursday, July 10th"

## Steps to reproduce

With voice assistant enabled for everything:

1. Single-tap on the Previous/Next month buttons

2. Double-tap on a date 

3. Double-tap on 2 different dates to select a range

4. Double-tap on 2 different dates to select a range

## OS / Browser / Device model

Tested on:
- Galaxy S8
- iPhone 7+

**Comment by charypar** [comment]
1. this is very tricky because it's a part of a 3rd party library. We can raise a PR to them to fix it.

for iOS:
2. is what VoiceOver does - tells you what you tapped (the date), then where you are including it's state (selected, date). I think that's intentional.
3. I really wanted to do this, but VoiceOver doesn't have live regions, it doesn't let you interrupt the user's flow to announce a change in a different part of the UI :-/

for Android:
4. That sounds like another bit of native behaviour, although I'm not as familiar with the Android one. 

When building this, I checked against the VoiceOver UX on the Calendar app in iOS as a bench mark and apart from the month navigation, we're essentially doing almost exactly the same thing, except for the selection. 

I'm afraid we can't really do much more than this :(

**Comment by RGBboy** [comment]
Should now be fixed by #378
